modules/API/external/external_api.js

```diff
@@ -1237,21 +1237,6 @@ export default class JitsiMeetExternalAPI extends EventEmitter {
         });
     }
 
-    /**
-     * Returns the state of availability electron share screen via external api.
-     *
-     * @returns {Promise}
-     *
-     * TODO: should be removed after we make sure that all Electron clients use only versions
-     * after with the legacy SS suport was removed from the electron SDK. If we remove it now the SS for Electron
-     * clients with older versions wont work.
-     */
-    _isNewElectronScreensharingSupported() {
```

Comment:
Why are we removing this? 

The support for this was recently removed in the electron-sdk (https://github.com/jitsi/jitsi-meet/commit/a3622a3698fe0578b87c753f735c017ebd945e7b). Shouldn't we wait for a bit the ^^^ commit to be released in the clients? If it is not released it seems that we are going to fallback to the old flow just because this function is missing....

Reply:
The old flow is gone, so there is no point in keeping the check any longer. Arguably this should have been made in  https://github.com/jitsi/jitsi-meet/commit/a3622a3698fe0578b87c753f735c017ebd945e7b because there are no longer 2 flows. It's just the new one.

Reply:
Alright, I restored this. We have a bit of a mess here so I got a bit confused. I think we need to bump the Electron SDK to 7 and then after a while to 8, where we only leave the new-new way of doing SS, using gDM and gDM only.